umbraco-heartcore/getting-started/webhooks.md

```diff
@@ -34,7 +34,7 @@ Should you at some point need to temporarily pause the webhook, you can disable
 
 Webhooks will be fired from either of the two static IPs listed below.
 
-When you are working with an external service that is behind a firewall and that service needs to communicate with your Umbraco Heartcore project in order to receive webhook data you need to make sure that the following IPs are allowed to pass through the firewall.
+When working with an external service behind a firewall, ensure the following IPs can pass through for webhook data. The service requires access to your Umbraco Heartcore project. 
```

Comment:
Leaving just "pass through for webhook data" might leave some questions so I added the part back where "the service needs to communicate heartcore project " Therefore my suggestions would be:
```suggestion
When working with an external service behind a firewall, your service needs to communicate with your Umbraco Cloud project and receive webhook data. This can be done by ensuring that the following IPs are allowed to pass from the firewall
```